src/api/status/src/js/feed-stats.js

```diff
@@ -3,9 +3,7 @@ const { fetch } = require('@senecacdot/satellite');
 module.exports = async function getFeedCount() {
   let feedCount = 0;
```

Comment:
Get rid of this and return on line 7 and again in the catch.

Reply:
I think if I do that I'm getting this 
![image](https://user-images.githubusercontent.com/58233223/145657876-f3e9b911-bc21-4fc5-bbc9-e57a877db566.png)
Same thing for getFeedCount()

Reply:
```js
  try {
    const data = await fetch(`${process.env.API_URL}/v1/posts/feeds`, {
      method: 'HEAD',
    });
    const data = await fetch(`${process.env.POSTS_URL}/feeds`, { method: 'HEAD' });
    return data.headers.get('x-total-count');
  } catch (err) {
    console.error(err);
    return 0;
  }
```